### ISSUE TITLE
About texture

### ISSUE THREAD

**Issue by winstywang** [ShapeNet/RenderForCNN#1]
Why the texture of models are not rendered?

**Comment by winstywang** [comment]
I find that only part of the models will not render the texture, quite wired.

**Comment by charlesq34** [comment]
While most models have textures, some models do not (e.g. bicycles).

On Mon, Jan 25, 2016 at 10:42 PM, Naiyan Wang notifications@github.com
wrote:

> I find that only part of the models will not render the texture, quite
> wired.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/ShapeNet/RenderForCNN/issues/1#issuecomment-174859319
> .

## 

## 

Charles Ruizhongtai Qi
PhD Candidate at Stanford AI Lab
Email: rqi@stanford.edu
Phone: (650)-804-3839

**Comment by winstywang** [comment]
Here is a render result for the same model using the provided codes and the results on shapenet website. 
Model ID: b96d8d6400e319eb8d6ee799b331d163

![bed1](https://cloud.githubusercontent.com/assets/4225664/12603625/5b3fd102-c4ef-11e5-8abf-8e866799dbe4.png)
![bed2](https://cloud.githubusercontent.com/assets/4225664/12603626/5b414226-c4ef-11e5-9c19-7e735b678d7b.png)

**Comment by bittnt** [comment]
Got puzzle here as well. Is this because the texture png image path are messed up or those texture png files are missing?

**Comment by winstywang** [comment]
@bittnt Did you meet the same situation? I have never modify the mtl file which defines the texture for the model.

**Comment by bittnt** [comment]
@winstywang Yes. I have also seen some textures are missing. Maybe because of the texture files are broken (not downloaded correctly) or missing.

**Comment by winstywang** [comment]
I tried to load it with other renderer, it works well. Is it a bug within blender, or do we need to load the texture in the codes?

**Comment by suhaochina** [comment]
Maybe you can try this renderer: https://github.com/shi-jian/mitsuba-shapenet

It works a lot better than blender for ShapeNet data. I haven't tried it for this specific model, though.

**Comment by HaoxuZhang** [comment]
@winstywang I met the same situation. Have you solved this problem?
